justfile

```diff
@@ -247,6 +247,41 @@ assets-test: assets-install
     npm run test:coverage
 
 
+# Build a lightweight local development setup using test fixture data.
+build-dbs-for-local-development nuclear="":
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # WARNING: Passing 'nuclear' to this just recipe will create a
+    # backup copy of your current local db, if it exists, then create a
+    # new empty core `db.sqlite3` in the root directory of your
+    # opencodelists project folder.
+    if [ -n "{{ nuclear }}" ]; then
+        echo "Nuclear option enabled: deleting db.sqlite3"
```

Comment:
instead of this, 
```
echo "Nuclear option enabled: moving $CORE_DB_PATH to $CORE_DB_PATH.backup"
```

after the following `if` block and before the `mv` would be the most informative